conf/application.conf

```diff
@@ -149,6 +149,7 @@ timeoutDialogCountdown = 120
 feature-switch {
   welshToggleEnabled = true
   nrsEnabled = true
+  tailoringEnabled = false
```

Comment:
this can actually be true here as long as it is false in app-base similar to Personal-frontend

Reply:
Updated